docker2earthly/convert.go

```diff
@@ -139,3 +140,38 @@ func Docker2Earthly(dockerfilePath, earthfilePath, imageTag string) error {
 	fmt.Fprintf(out, "\nbuild:\n    BUILD +subbuild%d\n", i)
 	return nil
 }
+
+func DockerWithEarthly(buildContextPath string, dockerfilePath, imageTag string, buildArgs []string, platforms []string, target string) error {
+	earthfilePath := filepath.Join(buildContextPath, "Earthfile")
+	defer os.Remove(earthfilePath)
```

Comment:
This `defer os.Remove` also seems wrong to me - this function writes the file and then removes it. I don't see a point where we actually run a build command before the `return` statement (and all of the `defer`red statements run immediately after the `return`).